devlib/native/win_native.cpp

```diff
@@ -121,6 +121,37 @@ namespace winutil {
     }
 
 
+    static auto initializeRootBuses(void) -> std::map<QString, int>
+    {
+        auto rootHubsBusesMap = std::map<QString, int>{};
+
+        auto deviceInfoSet = ::SetupDiGetClassDevs(nullptr, nullptr, nullptr, DIGCF_ALLCLASSES | DIGCF_PRESENT | DIGCF_DEVICEINTERFACE);
+        if (deviceInfoSet == INVALID_HANDLE_VALUE)
+            return {};
+
+        auto busNumber = 1;
+        DWORD index = 0;
```

Comment:
I think it's clear that `busNumber` is just `index+1`, so I don't think that there's a point in having a separate variable for this

Reply:
I don't agree with you.

I'm increasing `_index` for each device. But `busNumber` I is only incremented for devices with `instanceId` and `GUID_DEVINTERFACE_USB_HOST_CONTROLLER` interface.